content/en/agent/autodiscovery/endpointschecks.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Running Endpoints Checks with Autodiscovery
+kind: documentation
+further_reading:
+- link: "/agent/autodiscovery"
+  tag: "Documentation"
+  text: "Main Autodiscovery documentation"
+- link: "/agent/autodiscovery/clusterchecks/"
+  tag: "Documentation"
+  text: "Cluster Checks documentation"
+- link: "/agent/kubernetes/cluster/"
+  tag: "Documentation"
+  text: "Cluster Agent documentation"
+---
+
+## How it Works
+
+The [Cluster Checks][1] feature offers the possibility to auto-discover and perform Checks on load-balanced cluster services (eg. Kubernetes services). The Endpoints Checks feature extends this mechanism to monitor Endpoints behind cluster services.
+
+The [Cluster Agent][2] holds the configurations and exposes them to node-based Agents so they can consume and convert them into Endpoints Checks.
+
+Endpoints Checks are consumed and scheduled by Agents based on same nodes with worker pods of the service.
+
+The Agents connect to the Cluster Agent every 10 seconds and retrieve the configurations to run, the Endpoints Checks collected will be submitted with service, pod and hostname tags.
+
+This feature is currently supported on Kubernetes for versions 6.9.0+ of the Agent, and versions 1.3.0+ of the Cluster Agent.
+
+
+## How to set it up
+
+### Cluster Agent setup
+
+This feature requires a running [Cluster Agent with the Cluster Checks feature enabled][3].
+
+### Agent setup
+
+Enable the `clusterchecks` and `endpointschecks` configuration providers on the Datadog **Host** Agent. This can be done in two ways:
```

Comment:
```suggestion
Enable the `clusterchecks` and `endpointschecks` configuration providers on the Datadog **Node** Agent. This can be done in two ways:
```